### ISSUE TITLE
Add standard actions to file control bar

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#9491]
### Steps to reproduce
1. open a only office file in web
2. available actions in file control bar is X (Clos)

### Expected behaviour

1. Save (if available)
2. 3-dots (see cernbox for good example)

### Actual behaviour
- only close is available

**Comment by pascalwengerter** [comment]
@tbsbdr could you provide a list of actions, ideally prioritized, that you'd expect to be placed inside the three-dot menu?

Technical background: The actions that are used within the three-dots-/context-menu and sidebar currently live within the `web-app-files` and have dependencies within it (sometimes multiple levels deep), but need to be moved to the `web-pkg` so they're available for other, non `files-app` apps

**Comment by kulmann** [comment]
> @tbsbdr could you provide a list of actions, ideally prioritized, that you'd expect to be placed inside the three-dot menu?
> 
> Technical background: The actions that are used within the three-dots-/context-menu and sidebar currently live within the `web-app-files` and have dependencies within it (sometimes multiple levels deep), but need to be moved to the `web-pkg` so they're available for other, non `files-app` apps

Not necessarily. Please have a chat with @dschmidt about this. Apps can now register extensions. So I think a valid approach would be that the files app registers all the file actions as extensions in the runtime. This also ensures that "file actions" are only available (but then system wide) if the files app is installed, which IMO is a good thing.
